client/src/components/J40Header.tsx

```diff
@@ -25,7 +25,7 @@ const J40Header = () => {
   const toggleMobileNav = (): void =>
     setMobileNavOpen((prevOpen) => !prevOpen);
 
-  const headerLinks = (flags: string[] | undefined) => {
+  const headerLinks = (flags: {[key: string] : any} | undefined) => {
```

Comment:
So flags are now key:value pair? Seems reasonable.

Reply:
Yep - should arguably have been them all along - 
example is `?flags=low_tiles=NAME`